apidoc/Titanium/UI/ProgressBar.yml

```diff
@@ -72,6 +72,16 @@ properties:
     summary: Progress bar text.
     type: String
 
+  - name: progressTintColor
+    summary: The color shown for the portion of the progress bar that is filled.
+    type: String
+    platforms: [iphone, ipad]
+
+  - name: trackTintColor
+    summary: The color shown for the portion of the progress bar that is not filled.
+    type: String
+    platforms: [iphone, ipad]
+
```

Comment:
Please add the line ``since: "5.2.0"`` on both properties to let our users know since which SDK version the properties will be available. Then we're good to go!